import_flow_events.py

```diff
@@ -0,0 +1,143 @@
+#
+# Script to import "flow event" metrics from S3 into redshift.
+#
+
+import os
+import json
+import time
+import datetime
+
+import postgres
+
+import boto.s3
+import boto.provider
+
+# Load config from disk,
+# and pull in credentials from the environment.
+
+with open("config.json") as f:
+    CONFIG = json.loads(f.read())
+
+if "aws_access_key_id" not in CONFIG:
+    p = boto.provider.Provider("aws")
+    CONFIG["aws_access_key_id"] = p.get_access_key()
+    CONFIG["aws_secret_access_key"] = p.get_secret_key()
+
+DB = "postgresql://{db_username}:{db_password}@{db_host}:{db_port}/{db_name}".format(**CONFIG)
+
+# Event data files are named like "flow-2016-02-15.csv"
+# and contain events for that day.
+
+EVENTS_BUCKET = "net-mozaws-prod-us-west-2-pipeline-analysis"
+EVENTS_PREFIX = "fxa-flow/data/"
+EVENTS_FILE_URL = "s3://" + EVENTS_BUCKET + "/" + EVENTS_PREFIX + "flow-{day}.csv"
+
+# We import each into is own table and maintain a UNION ALL view over them.
+
+Q_CREATE_TABLE = """
+    CREATE TABLE IF NOT EXISTS flow_events (
+      timestamp BIGINT NOT NULL sortkey, 
```

Comment:
It's mainly for "I couldn't be bothered changing it reasons", which are pretty crappy reasons :-)

We've going to have to revisit how the activity event imports are handled anyways, since they are more frequent and include more data.  So I'll happily take your guidance on the right types etc here.  Might be easiest if you just do them as a PR to this PR?